DUI3-DX/Sdk/Speckle.Connectors.Utils/Operations/ReceiveOperation.cs

```diff
@@ -37,21 +37,20 @@ public ReceiveOperation(IHostObjectBuilder hostObjectBuilder)
       ?? throw new SpeckleAccountManagerException();
 
     // 3 - Get commit object from server
-    Client apiClient = new(account);
+    using Client apiClient = new(account);
     ServerTransport transport = new(account, projectId);
     Commit? version =
-      await apiClient.CommitGet(projectId, versionId, cancellationToken).ConfigureAwait(false)
+      await apiClient.CommitGet(projectId, versionId, cancellationToken).ConfigureAwait(true)
```

Comment:
this is a common purpose function, not sure this will be the case with other connectors?

Reply:
Good point.
`Operations.Receive` doesn't need to be done on the main thread.
Perhaps we should be syncing to main thread much further down the call sack (inside the `hostObjectBuilder`)

Reply:
Changed to using ISyncToMainThread